app/helpers/templates_helper.rb

```diff
@@ -28,6 +28,20 @@ def infobox_functions_and_macros(docs_section)
           :rel => 'external')
   end
 
+  def safemode_methods
```

Comment:
Please add a unit test for this method under test/unit/helpers/ to check it returns some known jail members.